app/assets/javascripts/covid19/src/data/LigandsApiRepository.ts

```diff
@@ -0,0 +1,89 @@
+import _ from "lodash";
+import { LigandImageData } from "../domain/entities/LigandImageData";
+import { LigandsRepository } from "../domain/repositories/LigandsRepository";
+import { routes } from "../routes";
+import { Future, FutureData } from "./utils/future";
+import { getJSON } from "./utils/request-utils";
+
+export class LigandsApiRepository implements LigandsRepository {
+    ligandsData: any[] = [];
+
+    constructor() {
+        getJSON<any[]>(`${routes.bionotesApi}/ligands/`).run(
```

Comment:
I've been calling /ligands/ endpoint because the API had no way to identify the ligand IDR with the information we had about the ligand. It was like a quick solution just to check the functionality of the components. By the way, thanks for the comment; I didn't know. I will try to do it as you tell me if I need to cache any variable.